docs/guide/custom-app.md

```diff
@@ -0,0 +1,225 @@
+          "preserveSymlinks": true,
+          "allowedCommonJsDependencies": [
+            "duration-relativetimeformat",
+            "papaparse",
+            "xlsx",
+            "chroma-js",
+            "@rgrove/parse-xml",
+            "@messageformat/core",
+            "rbush",
+            "@camptocamp/ogc-client",
+            "pbf",
+            "alasql"
+            // add dependencies here if other warnings show up and you want to hide them
+          ]
+        }
+      },
+      "defaultConfiguration": "production"
+    },
+  }
+```
+
+The `preserveSymlinks` setting is also important if you're working in dev mode and use a symbolic link to point
+to a dev build of GeoNetwork-UI.
+
+### Step 5: Install Tailwind
+
+[Tailwind CSS](https://tailwindcss.com/) is used for styling across the whole of GeoNetwork-UI, and is a mandatory dependency.
+
+To install and initialize it:
+
+```shell
+npm install -D tailwindcss postcss autoprefixer
+npx tailwindcss init
+```
+
+Note that this is taken from the Tailwind CSS setup guide.
+
+Then, adjust the `tailwind.config.js` file like so:
+
+```js
+const baseConfig = require('geonetwork-ui/tailwind.base.config.js')
+
+/** @type {import('tailwindcss').Config} */
+export default {
+  ...baseConfig,
+  content: ['./node_modules/geonetwork-ui/**/*.mjs', './src/**/*.{html,ts}'],
+}
+```
+
+Here we are inheriting from the GeoNetwork-UI base Tailwind config, which provides many essential things like theme colors etc.
+
+### Step 6: Install other mandatory dependencies
+
+[Angular Material](https://material.angular.io/) and [ngx-translate](https://github.com/ngx-translate/core) are other dependencies essential for many GeoNetwork-UI components. To install them:
+
+```shell
+npm install --save @angular/material @angular/material-moment-adapter @ngx-translate/core @ngx-translate/http-loader
+```
+
+### Step 7: Include the required fonts
+
+The root `index.html` file of your application should include the Material Symbols font for icons. Add these lines to itS `<head>` section:
+
+```html
+<head>
+  <!-- ... -->
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+  />
+</head>
+```
+
+### Step 8: Include the GeoNetwork-UI stylesheet
+
+GeoNetwork-UI comes with its own stylesheet, which you should include at the top of your application `style.css` file like so:
+
+```css
+@import 'geonetwork-ui/style.css';
+```
+
+### Step 9: Initialize the color theme
+
+GeoNetwork-UI lets users define their own theme based on primary and secondary colors for instance. By default, no theme is specified
+so UI components might not render properly. To define a theme, call the following function on the application module constructor:
+
+```ts
+export class AppModule {
```

Comment:
When generating an angular app with the CLI 17 I noticed, that by default the app is generated as standalone without any `AppModule`. Maybe the doc should mention where the color theme should be initialized for standalone apps or that the app should be generated with the `--no-standalone` flag?

Reply:
Very good point, I'll mention this in the doc; I don't think we can support standalone apps?